msi/rust.wxs

```diff
@@ -149,6 +154,10 @@
         <!-- Specifies a single cab file to be embedded in the installer's .msi. -->
         <MediaTemplate EmbedCab="yes" CompressionLevel="high" />
 
+        <!-- Send a WM_SETTINGCHANGE message to tell processes like explorer to update their
+             environments so any new command prompts get the updated %PATH% -->
+        <CustomActionRef Id="WixBroadcastEnvironmentChange" />
```

Comment:
In theory that should be enough but as it says on that page "No message is sent if a restart of the system is pending.". Also on my Window 10 install it never updates the environment when I install but after this change it does so maybe I have a broken Windows install or something.